jsonnet/telemeter/rules.libsonnet

```diff
@@ -204,6 +204,22 @@
                 max by (_id, managed_cluster_id) (acm_managed_cluster_worker_cores:max)
               |||,
             },
+            {
+              // Hypershift Cluster vCPU-hours for the last hour.
```

Comment:
HyperShift == ROSA HCP? Since we mention "ROSA HCP" below, it might be good to use the same terminology here.

Reply:
Well that's a tricky question.
Telemeter "hosts" metrics for all kind of clusters, including unmanaged clusters.
For this reason, the comment on top of `cluster:usage:workload:capacity_virtual_cpu_hours` definition is not saying that it concerns ROSA classic only... Because in fact this recording rule is defined and takes values for all kind of clusters, including ROSA classic but also OSD clusters or even OCP clusters installed on customer premises.
The "sister" `hostedcluster:hypershift_cluster_vcpus:vcpu_hours` recording rule suffers from the same fate: it is defined on cluster running the hypershift operator. This includes "our" management clusters hosting the ROSA HCP clusters but this could include as well any customer cluster running the hypershift operator.
So Hypershift != ROSA HCP.
With that in mind, this raises the concern that recording rule `rosa:cluster:vcpu_hours` is maybe poorly named. Indeed this rule is defined for:
- OSD clusters which are not ROSA
- OCP clusters installed and managed by the customers
- Clusters hosted by Hypershift but which are managed by the customer.

Reply:
FWIW, swatch is a single consumer of this metric, and we can easily handle whatever name is most clear to others. So I'm inclined to name precisely based on what the metric is, rather than its intended use.